src/tests/utils/ApiTestSteps.ts

```diff
@@ -29,117 +29,139 @@ HARNESS_API_INSTANCE.interceptors.request.use(awsSigv4Interceptor);
 const xmlParser = new XMLParser();
 const ajv = new Ajv({ strictTuples: false });
 
-export async function startStubServiceAndReturnSessionId(): Promise<any> {
-	const stubResponse = await stubStartPost();
+export async function startStubServiceAndReturnSessionId(journeyType: string): Promise<any> {
+	const stubResponse = await stubStartPost(journeyType);
 	return sessionPost(stubResponse.data.clientId, stubResponse.data.request);
 }
 
-export async function stubStartPost():Promise<any> {
+export async function stubStartPost(journeyType: string): Promise<any> {
 	const path = constants.DEV_IPV_STUB_URL;
-	const postRequest = await axios.post(`${path}`);
+
+	let postRequest: AxiosResponse;
+	switch (journeyType) {
+		case "NO_PHOTO_ID":
+			postRequest = await axios.post(`${path}`, { context: "bank_account" });
+			break;
+		case "INVALID":
+			postRequest = await axios.post(`${path}`, { context: "INVALID" });
+			break;
+		default:
+			postRequest = await axios.post(`${path}`);
+			break;
+	}
+
 	expect(postRequest.status).toBe(201);
 	return postRequest;
 }
 
-export async function sessionPost(clientId?: string, request?: string):Promise<any> {
+export async function sessionPost(clientId?: string, request?: string): Promise<any> {
 	const path = "/session";
 	try {
 		const postRequest = await API_INSTANCE.post(path, { client_id: clientId, request });
-		expect(postRequest.status).toBe(200);
 		return postRequest;
 	} catch (error: any) {
 		console.log(`Error response from ${path} endpoint: ${error}`);
 		return error.response;
-	} 
+	}
 }
 
-export async function claimedIdentityPost(givenName: any, familyName: any, dob: any, sessionId?: any):Promise<any> {
+export async function sessionConfigGet(sessionId: string): Promise<any> {
+	const path = "/session-config";
+	try {
+		const getRequest = await API_INSTANCE.get(path, { headers: { "x-govuk-signin-session-id": sessionId } });
+		return getRequest;
+	} catch (error: any) {
+		console.log(`Error response from ${path} endpoint: ${error}`);
+		return error.response;
+	}
+}
+
+export async function claimedIdentityPost(givenName: any, familyName: any, dob: any, sessionId?: any): Promise<any> {
```

Comment:
I know this wasn't you but could you please update these types? They all look like strings so should be a quick improvement